### ISSUE TITLE
CSS formatter removes useful space

### ISSUE THREAD

**Issue by anthonyvdotbe** [beautify-web/js-beautify#2024]
# Description
In the example below, formatting removes a useful space character: the output is less readable & introduces an asymmetry.

# Input
The code looked like this before beautification:
```
bar::before {
    content: "(" counter(foo) ")";
}
```

# Expected Output
The code should have looked exactly the same after beautification.

# Actual Output
After beautification, the space between `"("` and `counter(foo)` is removed (but the space between counter(foo) and ")" isn't, causing an asymmetry):
```
bar::before {
    content: "("counter(foo) ")";
}
```

# Steps to Reproduce
Format the input.

## Environment
OS: Windows

## Settings
VS Code 1.66.1 with default settings. Not sure how to extract the settings VS Code uses under the hood. Originally reported as https://github.com/microsoft/vscode/issues/147060

**Comment by mhnaeem** [comment]
Related to the following, should be fixed in the upcoming release. Maybe you can try reproducing on `main` branch

Issues:
- https://github.com/beautify-web/js-beautify/issues/1798
- https://github.com/beautify-web/js-beautify/issues/1817

PRs:
- https://github.com/beautify-web/js-beautify/pull/2020
- https://github.com/beautify-web/js-beautify/pull/2018

**Comment by bitwiseman** [comment]
Released 1.14.3 today.  This is fixed in that version.  Thanks @mhnaeem !
